spec/rails_helper.rb

```diff
@@ -25,7 +25,6 @@
 Dir[Rails.root.join("spec/support/**/*.rb")].sort.each { |f| require f }
 
 RSpec.configure do |config|
-  config.use_transactional_fixtures = true
```

Comment:
Any particular reason this line was deleted?

Reply:
I removed this by mistake. Thanks for spotting it 👍